src/lib/application/application.factory.ts

```diff
@@ -41,6 +41,9 @@ function transform(options: ApplicationOptions): ApplicationOptions {
   target.name = resolvePackageName(target.name.toString());
   target.version = !!target.version ? target.version : DEFAULT_VERSION;
 
+  target.caseNaming = !!target.caseNaming ? target.caseNaming : 'kebab-or-snake';
```

Comment:
I had to add this to fix tests. Not sure why it's required, since it's used by the `ControllerFactory` and not here. I would like someone to give a look at this.